README.md

```diff
@@ -139,6 +139,18 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
         ...
     </sly>
     ```
+    
+  - [3.4](#3.4) <a name='3.4'></a> **Always provide i18n translation hints.**
+  
+  The more context the translators have about a specific text, the better they can do their work.
```

Comment:
Providing a hint has some implications on the actual translation: 
If hints are leveraged there is no fallback to a dictionary entry without any hint, so for translations each used combination of text and hint must be translated (because the hint is actually part of the key).
That means you cannot e.g. reuse existing translations (being provided by the product).